yuglu-frontend/src/sign-up.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+
+export default class SignUpPage extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {};
+    }
+
+    handleInputChange(event) {
+        const target = event.target;
+        const name = target.name;
```

Comment:
these arent used anymore

Reply:
Surely const target is being used? I can remove const name tho.